.github/workflows/release.yaml

```diff
@@ -44,48 +40,64 @@ jobs:
+          CONTAINER=$(docker container ls --no-trunc --format "{{json . }}" | jq ' . | select(.Image|match("alerting-endpoint"))')
+          CONTAINER_ID=$(echo ${CONTAINER} | jq -r .ID)
+          CONTAINER_NETWORK=$(echo ${CONTAINER} | jq -r .Networks)
+          SEARCH_PATH=.[0].NetworkSettings.Networks.${CONTAINER_NETWORK}.IPAddress
+          ALERTING_ENDPOINT_IP=$(docker container inspect ${CONTAINER_ID} | jq -r ${SEARCH_PATH})
+          echo ALERTING_ENDPOINT_IP=${ALERTING_ENDPOINT_IP} >> $GITHUB_ENV
+      - name: Configure Connaisseur
         run: |
-          DOCKER_CONTENT_TRUST=1 docker pull $(yq e '.deployment.image' helm/values.yaml)
+          if [[ ! (${{ matrix.integration-test-arg }} == 'pre-config') ]]; then
+            yq eval -i 'del(.deployment)' tests/integration/update.yaml
+            envsubst < tests/integration/update.yaml > update
+            yq eval-all --inplace 'select(fileIndex == 0) * select(fileIndex == 1)' helm/values.yaml update
+          fi
+          echo "::group::values.yaml"
+          yq e '.' helm/values.yaml
+          echo "::endgroup::"
         shell: bash
-      - name: Remove Pull Policy Update
+      - name: Run actual integration test
         run: |
-          yq eval -i 'del(.deployment)' tests/integration/update.yaml
-          yq eval -i 'del(.deployment)' tests/integration/namespaced-update.yaml
-      - uses: ./.github/actions/integration-test
-        with:
-          integration-test-script: ${{ matrix.integration-test-script }}
+          bash tests/integration/integration-test.sh "${{ matrix.integration-test-arg }}"
+        shell: bash
 
   publish_chart:
     runs-on: ubuntu-latest
     needs: [version-match, integration-test]
     steps:
-    - name: Install helm and git
-      run: |
-        curl https://baltocdn.com/helm/signing.asc | sudo apt-key add -
-        sudo apt-get install apt-transport-https --yes
-        echo "deb https://baltocdn.com/helm/stable/debian/ all main" | sudo tee /etc/apt/sources.list.d/helm-stable-debian.list
-        sudo apt-get update
-        sudo apt-get install helm git
-    - name: Install yq
-      run: sudo snap install yq
-    - uses: actions/checkout@v2
-      with:
-        fetch-depth: 0
-    - name: Lint helm chart
-      run: helm lint helm
-    - name: Package and upload helm chart
-      run: |
-        git config user.name "versioning_user"
-        git config user.email "connaisseur@securesystems.dev"
-        CHART_VERSION=$(yq e '.version' helm/Chart.yaml)
-        helm package helm
-        mkdir -p charts
-        mv connaisseur*.tgz ./charts
-        git checkout gh-pages
-        cd charts
-        helm repo index . --url https://sse-secure-systems.github.io/connaisseur/charts
-        cd ..
-        git add ./charts
-        git commit -m "Publish helm chart ${CHART_VERSION}"
-        git push https://${{ secrets.GITHUB_TOKEN }}@github.com/sse-secure-systems/connaisseur.git
+      - name: Install helm and git
```

Comment:
Since we "Configure Connaisseur" this should be "Install Helm and Git", same for "Lint Helm chart" and potentially more. Or we go for no names in yml chart and go for "Configure connaisseur"
Idc, but consistency

Reply:
hey that's unfair. i just did the formatting here ... will apply changes though ....